code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,328 @@
+	slot_flags = SLOT_BELT
+	hitsound = 'sound/hallucinations/growl1.ogg'
+
+/obj/item/weapon/nullrod/claymore/chainsaw_sword
+	icon_state = "chainswordon"
+	item_state = "chainswordon"
+	name = "sacred chainsaw sword"
+	desc = "Suffer not a heretic to live."
+	slot_flags = SLOT_BELT
+	attack_verb = list("sawed", "torn", "cut", "chopped", "diced")
+	hitsound = 'sound/weapons/chainsaw.ogg'
+
+/obj/item/weapon/nullrod/claymore/glowing
+	icon_state = "swordon"
+	item_state = "swordon"
+	name = "force weapon"
+	desc = "The blade glows with the power of faith. Or possibly a battery."
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/nullrod/claymore/katana
+	name = "hanzo steel"
+	desc = "Capable of cutting clean through a holy claymore."
+	icon_state = "katana"
+	item_state = "katana"
+	slot_flags = SLOT_BELT | SLOT_BACK
+
+/obj/item/weapon/nullrod/claymore/saber
+	name = "light energy sword"
+	hitsound = 'sound/weapons/blade1.ogg'
+	icon_state = "swordblue"
+	item_state = "swordblue"
+	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/nullrod/claymore/saber/red
+	name = "dark energy sword"
+	icon_state = "swordred"
+	item_state = "swordred"
+	desc = "Woefully ineffective when used on steep terrain."
+
+/obj/item/weapon/nullrod/sord
+	name = "\improper UNREAL SORD"
+	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
+	icon_state = "sord"
+	item_state = "sord"
+	slot_flags = SLOT_BELT
+	force = 4.13
+	throwforce = 1
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/scythe
+	icon_state = "scythe0"
+	item_state = "scythe0"
+	name = "reaper scythe"
+	desc = "Ask not for whom the bell tolls..."
+	w_class = 4
+	armour_penetration = 35
+	slot_flags = SLOT_BACK
+	sharp = 1
+	edge = 1
+	attack_verb = list("chopped", "sliced", "cut", "reaped")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/weapon/nullrod/scythe/vibro
+	icon_state = "hfrequency0"
+	item_state = "hfrequency1"
+	name = "high frequency blade"
+	desc = "Bad references are the DNA of the soul."
+	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
+
+/obj/item/weapon/nullrod/hammmer
+	icon_state = "hammeron"
+	item_state = "hammeron"
+	name = "relic war hammer"
+	desc = "This war hammer cost the chaplain fourty thousand space dollars."
+	slot_flags = SLOT_BELT
+	w_class = 5
+	attack_verb = list("smashed", "bashed", "hammered", "crunched")
+
+/obj/item/weapon/nullrod/chainsaw
+	name = "chainsaw hand"
+	desc = "Good? Bad? You're the guy with the chainsaw hand."
+	icon_state = "chainsaw1"
+	item_state = "mounted_chainsaw"
+	w_class = 5
+	flags = NODROP | ABSTRACT
+	sharp = 1
+	edge = 1
+	attack_verb = list("sawed", "torn", "cut", "chopped", "diced")
+	hitsound = 'sound/weapons/chainsaw.ogg'
+
+/obj/item/weapon/nullrod/clown
```

Comment:
http://www.ss13.eu/tgdb/tg/latest_stats.html#chaplainweapon

Still nowhere near the most popular though